artifactory/commands/curl/curl.go

```diff
@@ -48,15 +48,29 @@ func Execute(args []string) error {
 	// Replace url argument with complete url.
 	command.Arguments[uriIndex] = targetUri
 
-	log.Debug(fmt.Sprintf("Executing curl command: '%s -u***:***'", strings.Join(command.Arguments, " ")))
-	// Add credentials flag to Command. In case of flag duplication, the latter is used by Curl.
-	credFlag := fmt.Sprintf("-u%s:%s", artDetails.User, artDetails.Password)
-	command.Arguments = append(command.Arguments, credFlag)
+	cmdWithoutCreds := strings.Join(command.Arguments, " ")
+	// Add credentials to curl command.
+	credentialsMessage, err := command.addCommandCredentials(artDetails)
 
 	// Run curl.
+	log.Debug(fmt.Sprintf("Executing curl command: '%s %s'", cmdWithoutCreds, credentialsMessage))
 	return gofrogcmd.RunCmd(command)
 }
 
+func (curlCmd *CurlCommand) addCommandCredentials(artDetails *config.ArtifactoryDetails) (string, error) {
```

Comment:
This function seems to never return an error, so we may need to remove the error as a returned value.